docs/UsersGuide/best/hub-pattern.md

```diff
@@ -14,3 +14,8 @@ typed ports are connected to typed ports using the Hub as an intermediary to get
 **Figure 9. Hub pattern.** Each hub instance is responsible for connecting to a remote node. Input port calls are
 repeated to corresponding output ports on a remote hub. These hubs have been demonstrated on Sockets,
 ARINC 653 Channels, High-speed hardware buses between nodes, and UARTs between nodes in an embedded system.
+
+## Generic Hub
+
+There is now a standard implementation of the hub pattern. The [GenericHub](../api/c++/html/svc_generic_hub.html) is an
```

Comment:
Is this a valid link?

Reply:
Yes, it appears within the generated documentation.  So, the link will become live once this is merged and autodocs run.